modules/serverless-applications-checking-revision-status.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * knative-serving/debugging-serverless-applications.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-applications-checking-revision-status_{context}"]
+= Checking Revision status
+
+You might need to check the status of your revision to identify the issue with your Serverless application.
+
+.Procedure
+
+. If you configure your route with a `Configuration` object, run the following command to get the name of the `Revision` object created for you deployment:
++
+[source,terminal]
+----
+$ oc get configuration <configuration-name> --output jsonpath="{.status.latestCreatedRevisionName}"
+----
++
+You can find the configuration name in the `Route.yaml` file.
++
+If you configure your route with revision directly, look up the revision name in the `Route.yaml` file.
+
+. Query for the status of the revision:
++
+[source,terminal]
+----
+$ oc get revision <revision-name> --output yaml
+----
```

Comment:
I'm noticing there are lots of different ways the steps are describing the `oc get` action across the procedures:

* Examine...
* Query...
* List...
* Check...
* See...
* Get...
* Verify...

Where possible (i.e., when it's a generic statement), consider syncing up on one of these styles for consistency.

Reply:
I went through all modules here and made a minor unification as you suggested. In most cases though, certain subtleties in the meaning would be lost, so I've kept those.